### ISSUE TITLE
Ran for 0 files

### ISSUE THREAD

**Issue by webmonch** [dzhus/flickr-takeaway-processor#1]
Hi,
I wanted to use your tool for a library of my photos (5400) that I downloaded from Flickr.
I put the pics in **media** folder and json into **meta** folder. But after I run the tool it says

[Info] 0/0 sidecar .json files parsed
[Info] Writing tags for 0 files
[Info] Exiftool sucessfully ran for 0 files

What might be the problem?

Thank you

**Comment by webmonch** [comment]
It also looks like downloaded images have different naming conventions and the only correct ids are in json files.

**Comment by minasm** [comment]
@mmvlad have you sorted the issue already?

**Comment by webmonch** [comment]
> @mmvlad have you sorted the issue already?

 unfortunately I don’t have time to do that :)

**Comment by MarcBoissonneault** [comment]
"Almost" same here.

TLDR;
I run the tool and it seems to find the sidecars .json files but then nothing!

My process:
_Note: I've first figured out that the README have minor errors._ 
Here are the steps to reproduce:
```
root@0eda40f19721:/photos# ls -1 media/ | wc -l
2067

root@0eda40f19721:/photos# ls -1 meta/ | wc -l
2067

root@0eda40f19721:/photos# flickr-takeway-processor media/ meta/
bash: flickr-takeway-processor: command not found
```

Wrong name for the command on the README, missing a `a` in `away`.

```
root@0eda40f19721:/photos# flickr-takeaway-processor media/ meta/
[Info] 0/0 sidecar .json files parsed
[Info] Writing tags for 0 files
[Info] Exiftool sucessfully ran for 0 files
```

Same error as mention in the current issue.
Oups! 
Let's look at the help!

```
root@0eda40f19721:/photos# flickr-takeaway-processor
Process Flickr takeaway files

Usage: flickr-takeaway-processor META MEDIA [-t|--threads THREADS]

Available options:
  -h,--help                Show this help text
  META                     Directory containing photo_*.json files
  MEDIA                    Directory with media files
  -t,--threads THREADS     Maximum number of threads for I/O-bound operations
```

Params are on the wrong order!

```
root@0eda40f19721:/photos# flickr-takeaway-processor meta/ media/
[Info] 0/2067 sidecar .json files parsed
[Info] Writing tags for 0 files
[Info] Exiftool sucessfully ran for 0 files

root@0eda40f19721:/photos# flickr-takeaway-processor meta/ media/ -t 1
[Info] 0/2067 sidecar .json files parsed
[Info] Writing tags for 0 files
[Info] Exiftool sucessfully ran for 0 files

```

That's where I'm stuck now.... 
I'm looking at the code, but I'm just lost since I don't know Haskell at all :P ...

**Comment by dzhus** [comment]
Woha I totally missed this whole thread – somewhy I had notifications switched off for my own repo 🙄 
Let me try to run it myself, haven't tried it for a while. And I'm still on Flickr :D

**Comment by dzhus** [comment]
> [Info] 0/2067 sidecar .json files parsed

Just tried a fresh takeout archive, the issue is definitely a change in Flickr sidecar format:

```
λ> parseSidecar' (fromText "/Users/dmitrydzhus/projects/flickr-takeaway-processor/meta/photo_14034151995.json")
Left "Error in $.geo: parsing Geo failed, expected Object, but encountered Array"
```

**Comment by dzhus** [comment]
This should no longer be a problem as of #3, please try a new version from master.
